pkg/cacheutil/memcached_client_test.go

```diff
@@ -522,25 +565,38 @@ func (c *memcachedClientBackendMock) Set(item *memcache.Item) error {
 	c.lock.Lock()
 	defer c.lock.Unlock()
 
+	c.setCount++
```

Comment:
It might be fine in a unit test... But do we need lock here? Default memcached client config has async concurrency > 1

Reply:
I think the default `lock` is sufficient, as nothing else accesses this variable.

Reply:
I don't know why I didn't see that lock. All good then